python/llm/example/langchain/native_int4/voiceassistant.py

```diff
@@ -47,6 +48,7 @@ def prepare_chain(args):
 
     # We use our BigdlNativeLLM to subsititute OpenAI web-required API
     callback_manager = CallbackManager([StreamingStdOutCallbackHandler()])
+                               
```

Comment:
maybe we should use transformer int4 for the llm for chat as well? @jason-dai

Reply:
yes, we want to do that if the performance is close